indico/modules/events/abstracts/templates/abstract/public.html

```diff
@@ -301,12 +301,30 @@ <h3 class="abstract-title">
                         </a>
                     {% endif %}
                 </div>
+            {% endif %}
+            {% if abstract.can_see_reviews(session.user) %}
+                <div class="group">
+                    <a class="i-button arrow icon-file-download dropdown-abstract-pdf"
+                       data-toggle="dropdown"></a>
+                    <ul class="dropdown">
+                        <li>
+                            <a href="{{ url_for('.display_abstract_pdf_export', abstract) }}"
+                               class="icon-file-pdf">PDF</a>
+                        </li>
+                        <li>
+                            <a href="{{ url_for('.manage_abstract_pdf_export', abstract) }}"
+                               class="icon-file-pdf">{% trans %}PDF with reviews{% endtrans %}</a>
```

Comment:
Maybe having no icons on the dropdown items would be the best option? All options are for PDF files anyway.